apps/testing-javascript/src/components/navigation.tsx

```diff
@@ -243,45 +243,39 @@ const MobileNav = () => {
         {status === 'unauthenticated' ? (
           <NavLink path="/login" label="Log in" />
         ) : null}
-        <li className="h-full">
-          <NavigationMenu.Root delayDuration={0} className="flex h-full">
-            <NavigationMenu.List className="flex h-full items-center justify-center">
-              <NavigationMenu.Item className="h-full">
-                <NavigationMenu.Trigger className="flex h-full items-center justify-center rounded-full px-2 py-2 hover:bg-gray-100">
-                  <MenuIcon className="h-5 w-5" aria-hidden />
-                  <span className="sr-only">Menu</span>
-                </NavigationMenu.Trigger>
-                <NavigationMenu.Content className="absolute left-0 top-full w-full border-t border-gray-100 bg-white shadow-2xl shadow-gray-500/20">
-                  <ul>
-                    {/* <MobileNavLink ... /> */}
-                    {status === 'authenticated' && (
```

Comment:
all content in this menu should be visible for looged in users only so I moved this condition on the top level